systemd/ceph-disk@.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Ceph disk activation: %f
+After=local-fs.target
+Wants=local-fs.target
+
+[Service]
+Type=oneshot
+KillMode=none
+Environment=CEPH_DISK_TIMEOUT=10000
+ExecStart=/bin/sh -c 'timeout $CEPH_DISK_TIMEOUT flock /var/lock/ceph-disk-$(basename %f) /usr/sbin/ceph-disk --verbose --log-stdout trigger --sync %f'
```

Comment:
I believe the logs displayed by "--log-stdout" won't be saved, right ? It would be interesting if we could log the messages to a single file